### ISSUE TITLE
Should not be  this repository private?

### ISSUE THREAD

**Issue by albertollamaso** [humancaresystems/You-i-enrollment#1]
@alexgm511 @jsilverHCS 

This repository is visible from any Github user that doesn't belongs to HCS since it is public.

Regards,

**Comment by jsilverHCS** [comment]
Yes I'd assume we want this private.  Wait for confirmation from Alex
before proceeding.

Thanks for identifying this, Alberto!

*Jared Silver <https://www.linkedin.com/pub/jared-silver/60/5a6/612>  *|  Vice
President, Technology & Data Analytics
*Human Care Systems <http://www.humancaresystems.com/>* | 617.297.7667
jsilver@humancaresystems.com
BERLIN | *BOSTON* | DUBAI | LONDON | MEXICO CITY | NEW YORK | PARIS |
PHOENIX | SAN ANTONIO | SAN FRANCISCO | SÃO PAULO | SHANGHAI | SYDNEY |
TOKYO | TORONTO


*Confidentiality Notice: This email may contain confidential information
that is intended only for the named recipient.  If **you received this
message in error, please delete the message and notify us immediately.
Thank you.*
